From ad8c49ce2b7bee9518c04951ca55f66dcd7dfe39 Mon Sep 17 00:00:00 2001
From: Paresh Bhagat <p-bhagat@ti.com>
Date: Tue, 26 May 2026 19:44:55 +0530
Subject: [PATCH] ti-apps-launcher: Remove pulseaudio-service dependency

Remove pulseaudio-service from RDEPENDS as we transition to PipeWire
support for audio functionality.

Signed-off-by: Paresh Bhagat <p-bhagat@ti.com>
---
 .../recipes-demos/ti-apps-launcher/ti-apps-launcher.bb           | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meta-ti-foundational/recipes-demos/ti-apps-launcher/ti-apps-launcher.bb b/meta-ti-foundational/recipes-demos/ti-apps-launcher/ti-apps-launcher.bb
index 12701f6..212fec7 100644
--- a/meta-ti-foundational/recipes-demos/ti-apps-launcher/ti-apps-launcher.bb
+++ b/meta-ti-foundational/recipes-demos/ti-apps-launcher/ti-apps-launcher.bb
@@ -34,7 +34,6 @@ RDEPENDS:${PN} = "\
     qt3d \
     bash \
     seva-launcher \
-    pulseaudio-service \
     qmltermwidget \
     qtdeclarative-qmlplugins \
     qtwayland-qmlplugins \
-- 
2.34.1

